pyproject.toml

```diff
@@ -69,7 +69,7 @@ openlineage = ["openlineage-integration-common!=1.15.0", "openlineage-airflow"]
 amazon = [
     "apache-airflow-providers-amazon[s3fs]>=3.0.0",
 ]
-google = ["apache-airflow-providers-google"]
+google = ["apache-airflow-providers-google>=10.11.0"]
```

Comment:
We should not add this lower bound since it would be a breaking change.

Cosmos still supports Airflow 2.4.0; in that case, users who currently install `astronomer-cosmos[google]` rely on `apache-airflow-providers-google<10.11.0`. They may only be interested in leveraging the BQ profile mapping and not Cloud Run, and they should still be allowed to install whichever version of the Google provider they want to.

Reply:
Of course, change commited.